plugins/.goreleaser.template.yaml

```diff
@@ -10,21 +10,11 @@ archives:
       - policies/**
 checksum:
   name_template: "checksums.txt"
-  ids:
```

Comment:
`ids` need to match the ones in `archives` or remove them to checksum everything